[LLVM][Docs][SPIRV] Correct `-spirv-ext` option name, reorder

Maetveis opened the pull request:
The option is `-spirv-ext` not `-spirv-extensions`. Also move the examples after the description of the option, instead of after the list of extensions, where its easy to miss when skimming.

Maetveis committed:
[LLVM][Docs][SPIRV] Correct `-spirv-ext` option name, reorder

The option is `-spirv-ext` not `-spirv-extensions`. Also move the
examples after the description of the option, instead of after the
list of extensions, where its easy to miss when skimming.

Keenuts commented on llvm/docs/SPIRVUsage.rst:
```diff
@@ -131,9 +131,20 @@ Extensions
+
+``-spirv-ext=+SPV_EXT_shader_atomic_float_add,+SPV_INTEL_arbitrary_precision_integers``
+
+To enable all extensions, use the following option:
+``-spirv-ext=all``
```
```suggestion
To enable all extensions, use the following option:
``-spirv-ext=all``
To enable all KHR extensions, use the following option:
``-spirv-ext=khr``
```

Keenuts reviewed (approved):
Thanks for the doc update! Just a small addition we could add

Maetveis committed:
Add an example on how to enable khronos extensions

Co-authored-by: Nathan Gauër <github@keenuts.net>

Maetveis committed:
Add blank line to correctly format.

reStructuredText requires blank lines to make a new paragraph. Otherwise
the text will be rendered as a continuation of the previous paragraph,
with no line break.
